graphql/user_resolver.go

```diff
@@ -29,7 +29,7 @@ func (r *userResolver) Patches(ctx context.Context, obj *restModel.APIDBUser, pa
 	apiPatches := []*restModel.APIPatch{}
 	for _, p := range patches {
 		apiPatch := restModel.APIPatch{}
-		if err = apiPatch.BuildFromService(p); err != nil {
+		if err = apiPatch.BuildFromService(p, false); err != nil {
```

Comment:
This also uses the same fragment on spruce as the query above, so spruce is currently fetching projectIdentifier here as well. 
[code](https://github.com/evergreen-ci/spruce/blob/main/src/gql/queries/user-patches.graphql#L7)